src/app/components/AddSource.svelte

```diff
@@ -3,6 +3,36 @@
   import ManualEntry from "./ManualEntry.svelte";
   import Upload from "./Upload.svelte";
 
+  // Split Button
+  const uploadSourcesFromFile = () => {
+    manualEntry = false;
+  };
+
+  const directEntrySources = () => {
+    manualEntry = true;
+  };
+
+  let splitBtns = [
+    {
+      color: "blue",
+      size: "medium",
+      text: "Upload",
+      isOutlined: false,
+      hasDropShadow: false,
+      dataCy: "add-sources-splitbtn-upload",
+      handleClick: uploadSourcesFromFile,
+    },
+    {
+      color: "grey",
+      size: "medium",
+      text: "Direct entry",
```

Comment:
Is this text supposed to be `Direct Entry `

Reply:
The button text has already been set to uppercase letter, so it will still display as DIRECT ENTRY 😁

Reply:
awesome! I have totally forgotten it, thank you for clarifying it